containers/flatpak/cockpit-client

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python3
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2018 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import subprocess
+import sys
+import gi
+
+gi.require_version("Gtk", "3.0")  # NOQA
+gi.require_version("WebKit2", "4.0")  # NOQA
+
+from gi.repository import Gtk, Gdk, WebKit2
+
+
+class Browser(Gtk.Window):
```

Comment:
Let's think about how we can avoid duplicating this code with cockpit-desktop (or even how to unify the scripts)

Reply:
The most obvious solution to me is to turn the webkit browser into a standalone python script that we can call from `cockpit-desktop`, as needed.